src/ramda/index.js

```diff
@@ -8,6 +8,7 @@ module.exports = r.omit([
   'flip',
   'forEach',
   'indexBy',
+  'isArrayLike',
```

Comment:
https://github.com/pnpm/ramda/blob/master/source/internal/_isArrayLike.js

Reply:
so yes, but that was also in ramda? why is it coming though now?